### PR TITLE
add icon generator

### PR DESCRIPTION
**Pull request by markdowney**

### CODE REVIEW

File: .baker/generators/app/templates/AppIcon.appiconset/Contents.json [hunk 1]
```diff
@@ -1,110 +1,82 @@
...
-        }
-    ],
-    "info":{
-        "version":1,
-        "author":"makeappicon"
+  "images": [
+    {
+      "size": "29x29",
+      "idiom": "iphone",
+      "filename": "-Small.png",
```
Comment (callmephilip) [inline .baker/generators/app/templates/AppIcon.appiconset/Contents.json]: these "-" file names look suspicious. is this intentional?

File: .baker/generators/app/templates/AppIcon.appiconset/Contents.json [hunk 2]
```diff
@@ -1,110 +1,82 @@
...
+    {
+      "size": "29x29",
+      "idiom": "iphone",
+      "filename": "-Small@3x.png",
+      "scale": "3x"
+    },
+    {
+      "size": "40x40",
+      "idiom": "iphone",
+      "filename": "-40@.png",
```
Comment (callmephilip) [inline .baker/generators/app/templates/AppIcon.appiconset/Contents.json]: is dangling "@" legit?

**Comment by callmephilip** [comment]
Seeing a problem with Android. To reproduce:
- npm install && npm run setup
- npm run android

It tries to build android app but fails with the following error:

```
* What went wrong:
Execution failed for task ':app:mergeDebugResources'.
> /Users/philip/projects/baker/app/android/app/src/main/res/WEB/ic_launcher.png:1:1: Error: Content is not allowed in prolog.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

**Comment by markdowney** [comment]
Should be fixed. Need to re-install mobile icon resizer:

```
npm install mobile-icon-resizer --force
```

**Pull request merged**